tests/integration/transactions/tester.cpp

```diff
@@ -100,20 +100,20 @@ TEST_F(BoltClient, DoubleRollbackWithoutTransaction) {
 TEST_F(BoltClient, DoubleBegin) {
   EXPECT_TRUE(Execute("begin"));
   EXPECT_THROW(Execute("begin", kNestedTransactions), ClientQueryException);
-  EXPECT_TRUE(TransactionActive());
+  EXPECT_FALSE(TransactionActive());
```

Comment:
Why did this change?

Reply:
Because from Bolt v3 `ACK_FAILURE` is deprecated, so after receiving a `FAILURE` message, we have to send `RESET` to the server, which will reset the server state, meaning aborting the ongoing transactions of that session.